packages/lesswrong/client/logging.ts

```diff
@@ -64,20 +66,23 @@ export function onUserChanged(user: UsersCurrent) {
 window.addEventListener('load', ev => {
   const urlParams = new URLSearchParams(document.location?.search)
 
-  captureEvent("pageLoadFinished", {
+  const eventPayload: Record<CamelCaseify<UtmParam, '_'>, string | null> & Record<string, AnyBecauseIsInput> = {
     url: document.location?.href,
     referrer: document.referrer,
     utmSource: urlParams.get('utm_source'),
     utmMedium: urlParams.get('utm_medium'),
     utmCampaign: urlParams.get('utm_campaign'),
     utmContent: urlParams.get('utm_content'),
     utmTerm: urlParams.get('utm_term'),
+    utmUserId: urlParams.get('utm_user_id'),
```

Comment:
This is useful to track because the user may be logged out at the time they follow the link